Feat/por/skos concept (#254)
Informasjonsforvaltning/fdk-rdf-parser: README.md

```
% nox -s tests -- -vv
```

## Creating a new release of the library
- Manually change the version in *pyproject.toml*.
- Optional: update other dependencies as well.
- Run `poetry update`

> - you will get a draft release suggestion after merging the changes to main branch on GitHub. Ref the "Release Drafter"-action.

Reviewer: ```suggestion
- All commits to the main branch on GitHub triggers the "Release Drafter"-action, it creates a release draft or adds the commit to an existing draft.
```

> - then click the **Releases** link and then the "Edit"-button (looks like a **pencil**) on the **Draft** version.

Reviewer: ```suggestion
- Click the **Releases** link on GitHub when you are ready to publish the draft, and then click the "Edit"-button (looks like a **pencil**) on the **Draft** version.
```

> - if version and tag number are the same, it is safe to click the **Publish release** button.

Reviewer: ```suggestion
- Check that the release-tag equals the version in [pyproject](https://github.com/Informasjonsforvaltning/fdk-rdf-parser/blob/main/pyproject.toml) before clicking the **Publish release** button. This triggers the **Release**-action, that uses poetry to release the new version to [PyPI](https://pypi.org/).
```

Reviewer: Er det slik vi legger til linker i markdown
